src/components/NewTable/components/TablePagination/components/TablePaginationButton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { tokens } from '../../../../../theme/tokens';
+
+interface TablePaginationArrowProps extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onClick'> {
+  active?: boolean;
+  onClick?: (page?: number) => void;
+}
+export const TablePaginationButton = ({ active, children, onClick, ...props }: TablePaginationArrowProps) => {
+  return (
+    <button
+      onClick={!!onClick ? () => onClick() : undefined}
+      className={cx(styles.root, { [styles.clickable]: !!onClick, [styles.active]: !!active })}
```

Comment:
Would clsx work for this? Since we have it installed.

Reply:
Yes, but cx comes from Emotion already thus making clsx not needed anymore

Reply:
🤔 but emotion/CSS was just added

Reply:
Mhm true story, this could be up for debate. I prefer using emotion/css over to emotion/react due to its possibility to just write normal css inside some backticks and not having to camelcase the different props.

Reply:
For the emotion thingy there are some articles online about it and a Stackoverflow thread that basically tells some differences:

`emotion/css:`
- use vanilla react className prop
- no custom jsx parser
- no babel config or file pragmas
- no extra typescript config
- generate class names outside of react components

`emotion/react:`
- zero config for @emotion/cache
- easy server side rendering
- easy access to theme through css function

https://stackoverflow.com/questions/57498130/what-is-the-benefit-of-emotion-react-over-emotion-css-for-a-react-project

We could talk about it :)